app/components/Header.tsx

```diff
@@ -0,0 +1,176 @@
+"use client";
+import { Link } from "@chakra-ui/next-js";
+import { Box, Grid, Stack } from "@chakra-ui/react";
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import { useState } from "react";
+
+export default function Header() {
+  const [hamburger, setHamburger] = useState(false);
+
+  const pathname = usePathname();
+
+  const handleHamburger = () => {
+    setHamburger(!hamburger);
+  };
+
+  return (
+    <Stack>
+      <Grid templateColumns={"500px 1fr 100px"}>
+        <Box p="12px">
+          <Image
+            src="/images/kenworthy-logo.jpg"
+            alt="Kenworthy Machine"
+            width={500}
+            height={500}
+          />
+        </Box>
+        <Box
+          display="flex"
+          alignItems="center"
+          justifyContent="flex-end"
+          gap="24px"
+        >
+          <Link
+            href="/"
+            color="white"
+            fontSize="24px"
+            borderBottom={
+              pathname === "/" ? "solid var(--primary-red) 2px" : "none"
+            }
+            _hover={{
+              color: "var(--text-color-light)",
+              borderBottom: "solid var(--primary-red) 2px",
+              transitionDuration: "400ms",
+            }}
+          >
+            Home
+          </Link>
+
+          <Link
+            href="/request-quote"
+            color="white"
+            fontSize="24px"
+            borderBottom={
+              pathname === "/request-quote"
+                ? "solid var(--primary-red) 2px"
+                : "none"
+            }
+            _hover={{
+              color: "var(--text-color-light)",
+              borderBottom: "solid var(--primary-red) 2px",
+              transitionDuration: "400ms",
+            }}
+          >
+            Request a Quote
+          </Link>
+
+          <div onClick={handleHamburger}>
+            <Image
+              src="/images/hamburger-menu.png"
+              alt="Hamburger Menu"
+              width={40}
+              height={40}
+            />
+          </div>
+        </Box>
+        <Box></Box>
```

Comment:
```suggestion
```

Reply:
It is meant to be a spacer so, for clarity, I created a styled component with Emotion at the bottom of the page called "Spacer" and replaced the Box element with the new Spacer element. It operates as a type of left margin for the Grid layout.

Reply:
Ah, got it. Thank you for clarifying!

Reply:
Oh this addresses my comment in my code. There's actually an in-built Chakra UI component called Spacer - https://chakra-ui.com/docs/components/flex 
If that meets the same needs I would suggest replacing your spacer component with that for consistency

Reply:
Done!